dockerfiles/dev-env-base/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Author Davide Baroli <davide.baroli@uni.lu>
```

Comment:
Please remove the list of authors. We do not follow this style and github easily shows the author of each file.